### DISCUSSION TITLE
Auto Arrange Clones on the bed

### DISCUSSION THREAD

**Discussion by Avpman2** [OrcaSlicer/OrcaSlicer#6126] [Q&A]
WOW, thanks for this!!!
I wonder if you could add one more tweak: auto-switch to the Device view tab once the code has been uploaded to the printer? The Orca-Flash does this and it's nice.

**Comment by lin-ycv** [comment]
Auto arrange clones would be nice, currently only the first clone is arranged, the rest just overlaps the original model/centered
![clone](https://github.com/user-attachments/assets/8a4d975c-167b-46b7-bdfa-c6c46c664534)


the option to auto-switch already exist
![image](https://github.com/user-attachments/assets/b32a6c84-f7e6-4655-87dd-3056240e650e)

**Reply by Avpman2** [reply]
It works for me. I just tried it with multiples and then cloning the multiples as well. I am using the latest nightly build.

**Comment by Aghost917** [comment]
How do you turn off Auto-arrange when cloning! It makes it very hard to clone something and then arrange something if it mess up my entire build plate, and then I have to start all over again.
